packages/test/local-server-stress-tests/package.json

```diff
@@ -0,0 +1,118 @@
+{
+	"name": "@fluid-internal/local-server-stress-tests",
```

Comment:
package seems like a better fit for \@fluid-private

Reply:
i thought about this, and will change it, but i also wasn't sure if we could run fluid private tests in our long running out stress runs

Reply:
I wouldn't expect issues with that